torch/nn/functional.py

```diff
@@ -4841,28 +4841,79 @@ def _in_projection(
 
 scaled_dot_product_attention = _add_docstr(
     torch._C._nn.scaled_dot_product_attention, r"""
+scaled_dot_product_attention(query, key, value, attn_mask=None, dropout_p=0.0, is_causal=False) -> Tensor:
+
 Computes scaled dot product attention on query, key and value tensors, using
 an optional attention mask if passed, and applying dropout if a probability
 greater than 0.0 is specified.
 
+.. code-block:: python
+
+    # Composite Implementation
```

Comment:
I don't think "Composite" is really a thing outside of our internal definition?
Maybe just say that this is an efficient implementation equivalent to the following.